configuration/configurator/DocumentationMarkup.hpp

```diff
@@ -0,0 +1,52 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#ifndef _DOCUMENTATION_MARKUP_HPP
+#define _DOCUMENTATION_MARKUP_HPP
+
+static const char* DM_SECT2 = "sect2";
```

Comment:
I like this approach, I think there was an XML Header used somewhere in our new code, which could be defined here instead.